hasher-matcher-actioner/webapp/src/Api.tsx

```diff
@@ -324,22 +324,29 @@ export async function fetchHashCount(): Promise<Response> {
 
 // TODO remove the trailing slash from the API URL, then add back the leading slash for /actions/ and /action-rules/ endpoints.
 
+// This class should be kept in sync with python class ActionPerformer
+type BackendActionPerformer = {
+  name: string;
+  config_subtype: string;
+  url: string;
+  headers: string;
+};
+
 type AllActions = {
   error_message: string;
-  actions_response: Array<{
-    name: string;
-    config_subtype: string;
-    url: string;
-    headers: string;
-  }>;
+  actions_response: Array<BackendActionPerformer>;
```

Comment:
Just define this inline because `BackendActionPerformer` isn't referenced anywhere else and it'll make it harder for someone to just export it later, defeating the point of your most recent changes.

Reply:
I want to keep it separate so its clear what needs to be kept in sync